Issue #31, tests updated

oduyemi opened the pull request.

awilkescom commented:
Hey @oduyemi 
Could you add a description of your change? And can you have a look at the failing tests and the conflict?

oduyemi commented:
The initial update for the pull request, was issue 31; landing page routing; for the landing page to display a welcome message rather than return a 404 status code. Then, the following updates:

Test written for "/'
 Renamed the function name experience to experience_by_index to get rid assertion errors in /resume-experience
Updated logic for /resume-skill and some other routes to eliminate errors
Updated test_pytest.py to ensure that all tests pass.
